.github/workflows/xdna-driver-pr-comment-ci.yml

```diff
@@ -0,0 +1,81 @@
+name: XDNA DRIVER PR COMMENT CI
+
+on:
+   issue_comment:
+    types: [created]
+
+jobs:
+   check:
+    runs-on: [ Ubuntu-22.04 ]
+    if: contains('amd-akshatah, kingtamamd, maxzhen, raphaelbamd, sonals, shirishjo ', format('{0},', github.actor)) && github.event.comment.body == '/build' && github.event.issue.pull_request
```

Comment:
Since this is security-related and a bit convoluted, is this safe. It is unclear.
For example, if I have a user `sonal` or `hisonals`, does it work?
Relying on string conversion looks scary to me.

Reply:
Agreed, Let me check if I can add a group instead of string with a change in this logic.